components/typography/styles.js

```diff
@@ -41,6 +41,27 @@ export const bodyStandardStyles = css`
 	}
 `;
 
+export const bodyCompactStylize = (tag) => css`
```

Comment:
Nit: I'd rename `tag` -> `selector` since technically it can be any CSS selector.